src/pybind/mgr/dashboard/ci/cephadm/start-cluster.sh

```diff
@@ -18,16 +18,16 @@ on_error() {
     if [ "$1" != "0" ]; then
         printf "\n\nERROR $1 thrown on line $2\n\n"
         printf "\n\nCollecting info...\n\n"
-        printf "\n\nDisplaying MGR logs:\n\n"
-        kcli ssh -u root -- ceph-node-00 'cephadm logs -n \$(cephadm ls | grep -Eo "mgr\.ceph[0-9a-z.-]+" | head -n 1) -- --no-tail --no-pager'
-        for vm_id in 0 1 2
+        printf "\nSaving MGR logs:\n"
+        mkdir -p ${CEPH_DEV_FOLDER}/logs
+        kcli ssh -u root -- ceph-node-00 'cephadm logs -n \$(cephadm ls | grep -Eo "mgr\.ceph[0-9a-z.-]+" | head -n 1) -- --no-tail --no-pager' > ${CEPH_DEV_FOLDER}/logs/mgr.cephadm.log
```

Comment:
Aren't these logs somehow sent to `journald`? Can't they be filtered with a `journalctl` command? Perhaps something like `journalctl -n all -ft cloud-init` would simplify a lot this expression.

Reply:
`journalctl -n all -ft cloud-init` displays the cephadm bootstrap logs of the ceph-node-00. What we want over here is the mgr container logs and its fetched using `cephadm logs` command.